src/tests/system/tests/test_access_control.py

```diff
@@ -0,0 +1,139 @@
+"""
+SSSD Authentication Test Cases
+
+:requirement: access control
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
```

Comment:
I don't think we need to run those both under 'root' and 'sssd'. Let's run under default service user only.
This will also allow to run tests against sssd-2.9

Reply:
Done

Reply:
You can remove this line and change the function to ```client.sssd.start()```